rules/opentitan/transform.bzl

```diff
@@ -40,7 +41,7 @@ def obj_transform(ctx, **kwargs):
             src.path,
             output.path,
         ],
-        executable = cc_toolchain.objcopy_executable,
+        executable = get_tool(cc_toolchain, "riscv32-unknown-elf-objcopy"),
```

Comment:
I think I'd still want the `endswith` (probably as part of `get_tool`) instead of specifying the full name. (Some builds include `-unknown` and some doesn't)

Reply:
I prefer the full name because most tools have very ambiguous endings, e.g. `as` is likely to pick up some random library instead, and even things like `clang` could choose either `clang` or `riscv32-unknown-elf-clang`. Those are probably the same, but you get the idea.